arch-wiki-docs.py

```diff
@@ -33,5 +34,12 @@
 
     downloader.download_images()
 
+    for lang_dir in next(os.walk(args.output_directory))[1]:
+        main_page = os.path.join(args.output_directory, lang_dir, 'Main_page.html')
+        index_html = os.path.join(args.output_directory, lang_dir, 'index.html')
+        if os.path.exists(main_page) and not os.path.islink(index_html):
```

Comment:
This should be `os.path.exists(index_html)` in order to prevent overwriting an existing file.